acceptence_test/main.tf

```diff
@@ -0,0 +1,378 @@
+terraform {
+  required_version = " >= 0.11.0"
```

Comment:
Please logs of the acceptance tests.

Reply:
Terraform 0.12  Test Results

```
Test Results using terraform 0.12

$ export TF_VAR_ULTRADNS_USERNAME=XXXXXXXXXXX
$ export TF_VAR_ULTRADNS_PASSWORD=XXXXXXXXXXX
$ export TF_VAR_ULTRADNS_BASEURL="https://restapi.test.ultradns.net:443"
$ terraform --version
Terraform v0.12.20

Your version of Terraform is out of date! The latest version
is 0.12.24. You can update by downloading from https://www.terraform.io/downloads.html
$ make testacc
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test  acceptence_test/acceptence_test.go  -v  -timeout 120m
=== RUN   TestUltraDNSPlugin
=== PAUSE TestUltraDNSPlugin
=== CONT  TestUltraDNSPlugin
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 retry.go:72: terraform [init -upgrade=false -no-color]
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:87: Running command terraform with args [init -upgrade=false -no-color]
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158: Initializing the backend...
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158: Initializing provider plugins...
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158: Terraform has been successfully initialized!
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158: You may now begin working with Terraform. Try running "terraform plan" to see
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158: any changes that are required for your infrastructure. All Terraform commands
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158: should now work.
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158: If you ever set or change modules or backend configuration for Terraform,
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158: rerun this command to reinitialize your working directory. If you forget, other
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:158: commands will detect it and remind you to do so if necessary.
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 retry.go:72: terraform [get -update -no-color]
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:87: Running command terraform with args [get -update -no-color]
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 retry.go:72: terraform [apply -input=false -auto-approve -lock=false -no-color]
TestUltraDNSPlugin 2020-04-27T16:43:40+05:30 command.go:87: Running command terraform with args [apply -input=false -auto-approve -lock=false -no-color]
TestUltraDNSPlugin 2020-04-27T16:43:41+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-minimal: Creating...
TestUltraDNSPlugin 2020-04-27T16:43:41+05:30 command.go:158: ultradns_rdpool.it: Creating...
TestUltraDNSPlugin 2020-04-27T16:43:41+05:30 command.go:158: ultradns_record.it: Creating...
TestUltraDNSPlugin 2020-04-27T16:43:41+05:30 command.go:158: ultradns_tcpool.it: Creating...
TestUltraDNSPlugin 2020-04-27T16:43:41+05:30 command.go:158: ultradns_record.foobar: Creating...
TestUltraDNSPlugin 2020-04-27T16:43:41+05:30 command.go:158: ultradns_tcpool.test-probe-ping-pool: Creating...
TestUltraDNSPlugin 2020-04-27T16:43:41+05:30 command.go:158: ultradns_dirpool.test-dirpool-maximal: Creating...
TestUltraDNSPlugin 2020-04-27T16:43:41+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-maximal: Creating...
TestUltraDNSPlugin 2020-04-27T16:43:41+05:30 command.go:158: ultradns_dirpool.test-dirpool-minimal: Creating...
TestUltraDNSPlugin 2020-04-27T16:43:45+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-minimal: Creation complete after 4s [id=testprobehttpminimal.com.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:45+05:30 command.go:158: ultradns_probe_http.test-probe-http-minimal: Creating...
TestUltraDNSPlugin 2020-04-27T16:43:46+05:30 command.go:158: ultradns_record.it: Creation complete after 4s [id=test-record-txt.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:46+05:30 command.go:158: ultradns_tcpool.it: Creation complete after 4s [id=1keufrf3.test.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:46+05:30 command.go:158: ultradns_record.foobar: Creation complete after 4s [id=terraform.1keufrf3.test.]
TestUltraDNSPlugin 2020-04-27T16:43:46+05:30 command.go:158: ultradns_rdpool.it: Creation complete after 5s [id=test-rdpool-minimal.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:46+05:30 command.go:158: ultradns_dirpool.test-dirpool-maximal: Creation complete after 5s [id=testdirpoolmaximal.com.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:46+05:30 command.go:158: ultradns_dirpool.test-dirpool-minimal: Creation complete after 5s [id=testdirpoolminimal.com.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:46+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-maximal: Creation complete after 5s [id=testprobehttpmaximal.com.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:46+05:30 command.go:158: ultradns_probe_http.test-probe-http-maximal: Creating...
TestUltraDNSPlugin 2020-04-27T16:43:46+05:30 command.go:158: ultradns_tcpool.test-probe-ping-pool: Creation complete after 5s [id=testprobepingpool.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:46+05:30 command.go:158: ultradns_probe_ping.it: Creating...
TestUltraDNSPlugin 2020-04-27T16:43:46+05:30 command.go:158: ultradns_probe_http.test-probe-http-minimal: Creation complete after 1s [id=060847CB4D0E3EB8]
TestUltraDNSPlugin 2020-04-27T16:43:47+05:30 command.go:158: ultradns_probe_http.test-probe-http-maximal: Creation complete after 1s [id=060847CB4D0E3EBC]
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: ultradns_probe_ping.it: Creation complete after 1s [id=060847CB4D0E3EBD]
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: Warning: Interpolation-only expressions are deprecated
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:   on main.tf line 7, in provider "ultradns":
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:    7:   username = "${var.ULTRADNS_USERNAME}"
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: Terraform 0.11 and earlier required all non-constant expressions to be
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: provided via interpolation syntax, but this pattern is now deprecated. To
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: silence this warning, remove the "${ sequence from the start and the }"
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: sequence from the end of this expression, leaving just the inner expression.
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: Template interpolation syntax is still used to construct strings from
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: expressions when the template includes multiple interpolation sequences or a
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: mixture of literal strings and interpolations. This deprecation applies only
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: to templates that consist entirely of a single interpolation sequence.
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: (and 2 more similar warnings elsewhere)
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: Warning: Quoted references are deprecated
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:   on main.tf line 127, in resource "ultradns_probe_ping" "it":
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:  127:   depends_on = ["ultradns_tcpool.test-probe-ping-pool"]
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: In this context, references are expected literally rather than in quotes.
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: Terraform 0.11 and earlier required quotes, but quoted references are now
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: deprecated and will be removed in a future version of Terraform. Remove the
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: quotes surrounding this reference to silence this warning.
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: (and 2 more similar warnings elsewhere)
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: Warning: Quoted type constraints are deprecated
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:   on variables.tf line 2, in variable "ULTRADNS_USERNAME":
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:    2:      type = "string"
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: Terraform 0.11 and earlier required type constraints to be given in quotes,
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: but that form is now deprecated and will be removed in a future version of
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: Terraform. To silence this warning, remove the quotes around "string".
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: (and 2 more similar warnings elsewhere)
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: Apply complete! Resources: 12 added, 0 changed, 0 destroyed.
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 retry.go:72: terraform [destroy -input=false -force -no-color]
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:87: Running command terraform with args [destroy -input=false -force -no-color]
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: ultradns_dirpool.test-dirpool-maximal: Refreshing state... [id=testdirpoolmaximal.com.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:48+05:30 command.go:158: ultradns_tcpool.test-probe-ping-pool: Refreshing state... [id=testprobepingpool.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:49+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-minimal: Refreshing state... [id=testprobehttpminimal.com.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:49+05:30 command.go:158: ultradns_rdpool.it: Refreshing state... [id=test-rdpool-minimal.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:49+05:30 command.go:158: ultradns_tcpool.it: Refreshing state... [id=1keufrf3.test.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:49+05:30 command.go:158: ultradns_dirpool.test-dirpool-minimal: Refreshing state... [id=testdirpoolminimal.com.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:49+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-maximal: Refreshing state... [id=testprobehttpmaximal.com.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:49+05:30 command.go:158: ultradns_record.it: Refreshing state... [id=test-record-txt.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:49+05:30 command.go:158: ultradns_record.foobar: Refreshing state... [id=terraform.1keufrf3.test.]
TestUltraDNSPlugin 2020-04-27T16:43:51+05:30 command.go:158: ultradns_probe_ping.it: Refreshing state... [id=060847CB4D0E3EBD]
TestUltraDNSPlugin 2020-04-27T16:43:52+05:30 command.go:158: ultradns_probe_http.test-probe-http-maximal: Refreshing state... [id=060847CB4D0E3EBC]
TestUltraDNSPlugin 2020-04-27T16:43:52+05:30 command.go:158: ultradns_probe_http.test-probe-http-minimal: Refreshing state... [id=060847CB4D0E3EB8]
TestUltraDNSPlugin 2020-04-27T16:43:53+05:30 command.go:158: ultradns_record.foobar: Destroying... [id=terraform.1keufrf3.test.]
TestUltraDNSPlugin 2020-04-27T16:43:53+05:30 command.go:158: ultradns_probe_ping.it: Destroying... [id=060847CB4D0E3EBD]
TestUltraDNSPlugin 2020-04-27T16:43:53+05:30 command.go:158: ultradns_tcpool.it: Destroying... [id=1keufrf3.test.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:53+05:30 command.go:158: ultradns_dirpool.test-dirpool-maximal: Destroying... [id=testdirpoolmaximal.com.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:53+05:30 command.go:158: ultradns_rdpool.it: Destroying... [id=test-rdpool-minimal.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:53+05:30 command.go:158: ultradns_dirpool.test-dirpool-minimal: Destroying... [id=testdirpoolminimal.com.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:53+05:30 command.go:158: ultradns_probe_http.test-probe-http-maximal: Destroying... [id=060847CB4D0E3EBC]
TestUltraDNSPlugin 2020-04-27T16:43:53+05:30 command.go:158: ultradns_probe_http.test-probe-http-minimal: Destroying... [id=060847CB4D0E3EB8]
TestUltraDNSPlugin 2020-04-27T16:43:53+05:30 command.go:158: ultradns_record.it: Destroying... [id=test-record-txt.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:55+05:30 command.go:158: ultradns_dirpool.test-dirpool-minimal: Destruction complete after 2s
TestUltraDNSPlugin 2020-04-27T16:43:56+05:30 command.go:158: ultradns_record.foobar: Destruction complete after 2s
TestUltraDNSPlugin 2020-04-27T16:43:56+05:30 command.go:158: ultradns_probe_ping.it: Destruction complete after 3s
TestUltraDNSPlugin 2020-04-27T16:43:56+05:30 command.go:158: ultradns_tcpool.test-probe-ping-pool: Destroying... [id=testprobepingpool.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:56+05:30 command.go:158: ultradns_record.it: Destruction complete after 3s
TestUltraDNSPlugin 2020-04-27T16:43:57+05:30 command.go:158: ultradns_probe_http.test-probe-http-maximal: Destruction complete after 3s
TestUltraDNSPlugin 2020-04-27T16:43:57+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-maximal: Destroying... [id=testprobehttpmaximal.com.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:57+05:30 command.go:158: ultradns_probe_http.test-probe-http-minimal: Destruction complete after 3s
TestUltraDNSPlugin 2020-04-27T16:43:57+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-minimal: Destroying... [id=testprobehttpminimal.com.1keufrf3.test]
TestUltraDNSPlugin 2020-04-27T16:43:57+05:30 command.go:158: ultradns_rdpool.it: Destruction complete after 3s
TestUltraDNSPlugin 2020-04-27T16:43:57+05:30 command.go:158: ultradns_dirpool.test-dirpool-maximal: Destruction complete after 3s
TestUltraDNSPlugin 2020-04-27T16:43:57+05:30 command.go:158: ultradns_tcpool.it: Destruction complete after 3s
TestUltraDNSPlugin 2020-04-27T16:43:57+05:30 command.go:158: ultradns_tcpool.test-probe-ping-pool: Destruction complete after 1s
TestUltraDNSPlugin 2020-04-27T16:43:57+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-maximal: Destruction complete after 1s
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-minimal: Destruction complete after 1s
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: Warning: Interpolation-only expressions are deprecated
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:   on main.tf line 7, in provider "ultradns":
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:    7:   username = "${var.ULTRADNS_USERNAME}"
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: Terraform 0.11 and earlier required all non-constant expressions to be
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: provided via interpolation syntax, but this pattern is now deprecated. To
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: silence this warning, remove the "${ sequence from the start and the }"
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: sequence from the end of this expression, leaving just the inner expression.
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: Template interpolation syntax is still used to construct strings from
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: expressions when the template includes multiple interpolation sequences or a
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: mixture of literal strings and interpolations. This deprecation applies only
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: to templates that consist entirely of a single interpolation sequence.
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: (and 2 more similar warnings elsewhere)
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: Warning: Quoted references are deprecated
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:   on main.tf line 127, in resource "ultradns_probe_ping" "it":
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:  127:   depends_on = ["ultradns_tcpool.test-probe-ping-pool"]
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: In this context, references are expected literally rather than in quotes.
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: Terraform 0.11 and earlier required quotes, but quoted references are now
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: deprecated and will be removed in a future version of Terraform. Remove the
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: quotes surrounding this reference to silence this warning.
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: (and 2 more similar warnings elsewhere)
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: Warning: Quoted type constraints are deprecated
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:   on variables.tf line 2, in variable "ULTRADNS_USERNAME":
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:    2:      type = "string"
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: Terraform 0.11 and earlier required type constraints to be given in quotes,
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: but that form is now deprecated and will be removed in a future version of
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: Terraform. To silence this warning, remove the quotes around "string".
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: (and 2 more similar warnings elsewhere)
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:43:58+05:30 command.go:158: Destroy complete! Resources: 12 destroyed.
--- PASS: TestUltraDNSPlugin (17.60s)
PASS
ok      command-line-arguments  17.606s

```